loadbalanced-fargate-svc/service/instance_infrastructure/cloudformation.yaml

```diff
@@ -22,23 +22,23 @@ Resources:
   LogGroup:
     Type: AWS::Logs::LogGroup
     Properties:
-      LogGroupName: '{{service_name}}/{{service_instance_name}}'
+      LogGroupName: '{{service.name}}/{{service_instance.name}}'
 
   # The task definition. This is a simple metadata description of what
   # container to run, and what resource requirements it has.
   TaskDefinition:
     Type: AWS::ECS::TaskDefinition
     Properties:
-      Family: '{{service_name}}_{{service_instance_name}}'
+      Family: '{{service.name}}_{{service_instance.name}}'
       Cpu: !FindInMap [TaskSize, {{service_instance.task_size}}, cpu]
       Memory: !FindInMap [TaskSize, {{service_instance.task_size}}, memory]
       NetworkMode: awsvpc
       RequiresCompatibilities:
         - FARGATE
-      ExecutionRoleArn: '{{environment.ECSTaskExecutionRole}}'
+      ExecutionRoleArn: '{{environment.outputs.ECSTaskExecutionRole}}'
       TaskRoleArn: !Ref "AWS::NoValue"
       ContainerDefinitions:
-        - Name: '{{service_instance_name}}'
+        - Name: '{{service_instance.name}}'
           Cpu: !FindInMap [TaskSize, {{service_instance.task_size}}, cpu]
```

Comment:
wouldn't {{ service_instance.<blah> }} change to {{ service_instance.inputs.<blah> }}

Reply:
not for the name.  `service_instance` has a `name` and `inputs` properties

Reply:
oh wait, I see it now, great catch